src/WebJobs.Script/ScriptConstants.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public static class ScriptConstants
+    {
+        public const string AzureFunctionsWebHookContextKey = "MS_AzureFunctionsWebHookContext";
+
+        public const string AzureFunctionsHttpResponseKey = "MS_AzureFunctionsHttpResponse";
+
+        public const string DefaultSystemTriggerParameterName = "<AF>systemTrigger";
```

Comment:
This name is going to show up in the Dashboard, along with all the other system parameters. Currently we also add a "context" system parameter, right? If we're going to have a naming convention for these, we should be consistent. Similarly for Node.js functions, we are adding some system params (log, binder, context) with no prefix currently.